xbmc/filesystem/NFSFile.cpp

```diff
@@ -461,6 +466,14 @@ void CNfsConnection::AddIdleConnection()
   m_IdleTimeout = 180;
 }
 
+
+void CNfsConnection::setTimeout(struct nfs_context *context)
+{
+  uint32_t timeout = CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()->m_nfsTimeout;
+  if (~timeout)
+    nfs_set_timeout(context, timeout);
```

Comment:
This will break compiling on Ubuntu Xenial (16.04) with enabled nfs support.

Reply:
Because the version does not support nfs_set_timeout ? If so, do we have the libnfs version somewhere during compile time?

Reply:
`nfs_set_timeout` is available since 2.0.0 (https://github.com/sahlberg/libnfs/commit/b01776cf9411f3d1b83950c90078683ef78b4411)
Ubuntu Xenial (16.04) provides 1.9.8 (https://packages.ubuntu.com/xenial/libnfs-dev)